src/components/PlayCard.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ambassador from '../images/ambassador.JPG';
+import assasin from '../images/assasin.JPG';
+import back from '../images/back.JPG';
+import captain from '../images/captain.JPG';
+import contessa from '../images/contessa.JPG';
+import duke from '../images/duke.JPG';
+
+export const all_chars = ["Duke", "Assassin", "Contessa", "Captain", "Ambassador"];
```

Comment:
This isn’t used right? If it is you should import from game_logic

Reply:
oops forgot to remove it